Core/Core/Models/Assignment.swift

```diff
@@ -107,6 +108,10 @@ extension Assignment {
             }
         }
 
+        if let apiRubricSettings = item.rubric_settings {
+            hideRubricPoints = apiRubricSettings.hide_points
+        }
```

Comment:
nitpick: this could be `hideRubricPoints = item.rubric_settings?.hide_points == true`